v-next/hardhat/src/internal/builtin-plugins/network-manager/rpc/types/address.ts

```diff
@@ -14,6 +14,7 @@ export const rpcAddress: ZodType<Uint8Array> = conditionalUnionType(
       z.instanceof(Uint8Array),
     ],
     [isAddress, z.string()],
+    [(data) => data === null, z.null()],
```

Comment:
wouldn't this also affect `from`?

Reply:
Yes, I allowed the null value only for the "to" field [here](https://github.com/NomicFoundation/hardhat/pull/6017/commits/756248a4fd0b0e3fdeb3e290e161f0697d71bb19)